frontend_server/lib/server.dart

```diff
@@ -192,35 +184,28 @@ class _FrontendCompiler implements CompilerInterface {
   Future<Null> recompileDelta() async {
     final String boundaryKey = new Uuid().generateV4();
     _outputStream.writeln("result $boundaryKey");
-    final DeltaProgram deltaProgram = await _generator.computeDelta();
+    final Program deltaProgram = await _generator.computeDelta();
     final IOSink sink = new File(_kernelBinaryFilename).openWrite();
     final BinaryPrinter printer = printerFactory.newBinaryPrinter(sink);
-    printer.writeProgramFile(deltaProgram.newProgram);
+    printer.writeProgramFile(deltaProgram);
     await sink.close();
     _outputStream.writeln("$boundaryKey $_kernelBinaryFilename");
     return null;
   }
 
-  @override
-  void acceptLastDelta() {
-    _generator.acceptLastDelta();
-  }
-
-  @override
-  void rejectLastDelta() {
-    _generator.rejectLastDelta();
-  }
-
   @override
   void invalidate(Uri uri) {
-    _generator.invalidate(uri);
+    _generator?.invalidate(uri);
```

Comment:
Why is this change needed? We don't expect `invalidate` being called without `_generator` in place, do we?

Reply:
Oops. Left over from the time when I was just nulling out _generator. Reverting